conf/evolutions/default/22.sql

```diff
@@ -27,4 +27,5 @@ CREATE TABLE Organization(
 # --- !Downs
 
 DROP FUNCTION uuid_of;
-DROP TABLE Organization;
\ No newline at end of file
+DROP TABLE Organization;
+DROP FOREIGN KEY FK_Profile_Organization;
```

Comment:
This seems to me SDL, but if you want to edit data on runtime,  you have to use the data manipulation language of SQL. I think it should be `CASCADE ON DELETE`